recipes/keyrings.alt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "keyrings.alt" %}
```

Comment:
Neither conda build nor conda install complains so I guess yes?

Is it supposed to be a conda-forge limitation?